1/bump_version.py

```diff
@@ -191,13 +191,15 @@ def update_version_in_file(version_file_path: str, new_version: str):
 
 def add_ticket(changelog_path, new_version):
     if not (ticket_base_url := os.getenv("PLUGIN_TICKET_URL")):
+        print(f"Skipping adding the ticket as no ticket url is provided.")
         return
 
     if not (ticket_key_pattern := os.getenv("PLUGIN_TICKET_PATTERN")):
         print(f"Skipping adding the ticket as no ticket pattern is provided.")
         return
```

Comment:
This should raise an exception as user is expecting a pattern to be provided if a URL is set.
```suggestion
        raise Exception("Skipping adding the ticket as no ticket pattern is provided.")
```